web/cobrands/northamptonshire/assets.js

```diff
@@ -21,32 +15,26 @@ var layers = [
   ],
   "item_name": "drain",
   "layer_name": "Gully",
-  "styleid": '5d480b8ffe2ad809d85a78ff',
   "max_resolution": 0.5971642833948135
 },
 {
   "categories": [ "Grit Bin - damaged/replacement", "Grit Bin - empty/refill" ],
   "item_name": "grit bin",
-  "layer_name": "Grit Bins",
-  "styleid": '5d480942fe2ad809d85a78ad',
+  "layer_name": "GritBins"
 },
 {
   "categories": [ "Highway Bridges - Damaged/Unsafe" ],
-  "asset_type": 'area',
```

Comment:
This could be left in, and then included in your loop if set to be set - is used just in some classes in the main code to not show a 'spot'.

Reply:
The bridges layer is actually a spot layer (and it seems it was always a spot layer coming from alloy). Does it still need this, even though it's a spot layer?

Reply:
Nah, no need if it actually is spot (sorry for not actually checking those two, assumed it'd be correct already!).